docs/src/examples.md

```diff
@@ -1,6 +1,85 @@
 # Examples
 
-## 1D Column examples
+### Advection
+
+The 1D Column advection example in [`examples/column/advect.jl`](https://github.com/CliMA/ClimaCore.jl/blob/main/examples/column/advect.jl).
+
+#### Equations and Discretizations
+
+Follows the advection equation
+
+```math
+\begin{equation}
+  \frac{\partial \theta}{\partial t} = -\frac{\partial (v \theta)}{\partial z}
```

Comment:
Please use proper punctuation to connect or end sentences, even after math formulas/equations.

```suggestion
  \frac{\partial \theta}{\partial t} = -\frac{\partial (v \theta)}{\partial z} .
```